### PR TITLE
Update Smart Contracts Use Case for style and consistency

### PR DESCRIPTION
**Pull request by DennisDawson**

### CODE REVIEW

File: docs/img/uc-yescrow-holding-lock-and-key.png
Comment (amarantha-k) [inline docs/img/uc-yescrow-holding-lock-and-key.png:1]: The crow doesn't seem like the right illustration, but I don't have a good alternative in mind. Would defer this one to UX/design.

**Review by amarantha-k — CHANGES_REQUESTED** [review]
As noted in the comment, I think we need an alternative to the crow for escrow.

File: docs/img/uc-yescrow-holding-lock-and-key.png
Comment (amarantha-k) [reply docs/img/uc-yescrow-holding-lock-and-key.png:1]: Tagging @bugsbunnies for input.

Comment (bugsbunnies) [reply docs/img/uc-yescrow-holding-lock-and-key.png:1]: Awesome illustrations! For the lock part, I would suggest making all the strokes of the lock green as well so both the lock and key stands out in green as highlighted items.

Comment (bugsbunnies) [reply docs/img/uc-yescrow-holding-lock-and-key.png:1]: Would be good to also match the bird's eyes with the previous bird's eyes for consistency, they look slightly different.

Comment (DennisDawson) [reply docs/img/uc-yescrow-holding-lock-and-key.png:1]: Fixed, thanks for the feedback. Still need to rethink the crow, though.

Comment (DennisDawson) [reply docs/img/uc-yescrow-holding-lock-and-key.png:1]: Okay, a more apt metaphor, I reckon. The Escrow is a locker, the condition is a padlock, the fulfillment is the key.

**Review by bugsbunnies — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]
Thanks for incorporating the feedback.

**Pull request merged**